src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,12 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
+        String clean = text.replaceAll("[\\W]", "").toUpperCase();
+        String reverseString = new StringBuffer(clean).reverse().toString();
+        System.out.println(reverseString);
```

Comment:
remove sout, you can use it to test yourself but don't push it to github)

Reply:
Good morning, well noted, will reply in the nearest. Thanks